lighthouse-core/lib/i18n/i18n.js

```diff
@@ -86,6 +86,8 @@ const UIStrings = {
   columnElement: 'Element',
   /** Label for a column in a data table; entries will be the number of milliseconds since the page started loading. */
   columnStartTime: 'Start Time',
+  /** Label for a column in a data table; entries will be the reasons why an animation cannot be composited. */
```

Comment:
just providing some background.


in general we move strings to the i18n file if we're gonna share them across multiple audits. otherwise.. the string will live in 1 file that uses it.

so for this.. i think it would make sense to keep it here only if the string and its description are generic.  so maybe

```js
  /** Label for a column in a data table; entries will be the reasons explaining a failure for that item. (e.g the reason why the browser couldn't optimize a DOM element) */
  columnReasons: 'Reasons',
```

(in this case i think we can drop the word "failure" without losing much)